src/base/utils/misc.cpp

```diff
@@ -696,3 +696,15 @@ QString Utils::Misc::libtorrentVersionString()
     static const QString ver = LIBTORRENT_VERSION;
     return ver;
 }
+
+#ifdef Q_OS_WIN
+QString Utils::Misc::windowsSystemPath()
+{
+    static const QString path = []() -> QString {
```

Comment:
Oops... I didn't recognize **static** keyword. Then it should be. But...
>I wanted to minimize calling  GetSystemDirectoryW().

Isn't it over-optimization?